### ISSUE TITLE
Create Group Members response does not hold groupId

### ISSUE THREAD

**Issue by nikos** [FusionAuth/fusionauth-issues#552]
### Description 
The groupId should be filled in the case of a succesful group membership assignment.

### Steps to reproduce

POST http://localhost:9011/api/group/member

### Expected behavior
groupId should be filled

### Screenshots
Will upload

### Platform
FusionAuth 1.15.5 in docker container
Latest Java FusionAuth API

**Comment by nikos** [comment]
Screenshot:
https://imgur.com/a/Q4EoNZC

(Github stated a problem when uploading the PNG, sorry)

**Comment by robotdan** [comment]
Can you post the JSON success response from the API? Your screenshot looks like you've highlighted the request object?  

Here is the example output from that API
https://fusionauth.io/docs/v1/tech/apis/groups#add-users-to-a-group

```
  "members": {
    "1188edfc-cef3-4555-910e-181ddf6153c0": [
      {
        "data": {
          "fruit": "orange"
        },
        "id": "47ffe8c2-920a-49cc-bfa8-b84889db615e",
        "insertInstant": 1503513015493,
        "userId": "00000000-0000-0001-0000-000000000000"
      }
    ]
  }
}
```

Is this not what you are seeing?

**Comment by nikos** [comment]
@robotdan Thanks for your answer. So in your example I guess `1188edfc-cef3-4555-910e-181ddf6153c0` is the UUID of the group?
I was expecting in the groupId Java DTO filled as opposed to be null, or am I still missing something?

**Comment by robotdan** [comment]
> Thanks for your answer. So in your example I guess `1188edfc-cef3-4555-910e-181ddf6153c0` is the UUID of the group?

Yes, that is correct. This is kind of a strange API, the response is a map keyed by the `groupId`. 

![image](https://user-images.githubusercontent.com/1148709/79000293-96042e80-7b09-11ea-9027-1885764ea270.png)

**Comment by nikos** [comment]
Thanks @robotdan for clarification.

Would you think providing in this case the map key also in the groupId field make sense (aka developer expierence/expectation)?

**Comment by robotdan** [comment]
@nikos yes - this API needs some work I think. Let's keep this issue open as a usability issue so we (FusionAuth) can review further.

**Comment by nikos** [comment]
Thanks a lot for picking it up
